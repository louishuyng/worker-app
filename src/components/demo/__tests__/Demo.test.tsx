import 'react-native';
import * as React from 'react';
import renderer from 'react-test-renderer';

import Demo from '../index';

const props = {
  navigation: {
    goBack: jest.fn(),
  },
};

const component = (
  <Demo {...props} />
);

describe('[Temp] render', () => {
  it('renders without crashing', () => {
    const rendered = renderer.create(component).toJSON();
    expect(rendered).toMatchSnapshot();
    expect(rendered).toBeTruthy();
  });
});
