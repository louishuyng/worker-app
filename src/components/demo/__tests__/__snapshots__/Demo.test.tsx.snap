// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Temp] render renders without crashing 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "alignItems": "center",
        "display": "flex",
        "height": "100%",
        "justifyContent": "center",
      },
    ]
  }
>
  <Text>
    DEMO
  </Text>
</RCTSafeAreaView>
`;
